### PR TITLE
Update to Jakarta dependencies

### PR DESCRIPTION
**Pull request by aguibert**

### CODE REVIEW

**Comment by aguibert** [comment]
hi @Verdent can you please review?

File: pom.xml [hunk 1]
```diff
@@ -487,76 +520,6 @@
         </plugins>
     </build>
 
-    <dependencyManagement>
```
Comment (Verdent) [inline pom.xml]: Why did you remove dependecy management?

File: pom.xml [hunk 2]
```diff
@@ -26,6 +26,54 @@
...
+        <jakarta.json.version>1.1.6</jakarta.json.version>
+        <netbeans.hint.jdkPlatform>JDK_9</netbeans.hint.jdkPlatform>
+    </properties>
+    
+    <dependencies>
+        <!-- Compile dependencies -->
+        <dependency>
+            <groupId>jakarta.json.bind</groupId>
+            <artifactId>jakarta.json.bind-api</artifactId>
+            <version>1.0.2</version>
```
Comment (Verdent) [inline pom.xml]: Why not to use property as before?

File: pom.xml (hunk 1 above)
Comment (aguibert) [reply pom.xml]: dependencyManagement doesn't make sense in this repo since we have no submodules to manage dependencies for. It just makes the pom more verbose with no real benefit

File: pom.xml (hunk 2 above)
Comment (aguibert) [reply pom.xml]: similar to above, since the property was only ever used in 1 place, there isn't really a point in having a property. If it was used in 2 or more places then it would make sense to use a property

**Comment by aguibert** [comment]
hi @Verdent, any further thoughts on this?

File: pom.xml (hunk 2 above)
Comment (aguibert) [reply pom.xml]: I'll swap it back to the way it was originally (with a property) since this one is just a preference vs. a preference
